optbinning/binning/continuous_binning.py

```diff
@@ -717,16 +717,14 @@ def _fit(self, x, y, sample_weight, check_input):
             max_x = None
 
         if self.dtype == "numerical":
-            monotonic_score_kendalltau, kendalltau_pvalue = kendalltau(x_clean, y_clean)
-            monotonic_score_spearmanr, spearmanr_pvalue = spearmanr(x_clean, y_clean)
+            correlation_kendalltau, kendalltau_pvalue = kendalltau(x_clean, y_clean)
```

Comment:
The variable ``kendalltau_pvalue`` is not used.

Reply:
Pass flake8.

Reply:
done ~